modules/rostests/apitests/appshim/dispmode.c

```diff
@@ -474,20 +474,20 @@ START_TEST(dispmode)
                 }
             }
 
-            ok(failures == winetest_get_failures(), "Last %u failures are from %d (%s)\n",
+            ok(failures == winetest_get_failures(), "Last %u failures are from %Iu (%s)\n",
                 winetest_get_failures() - failures, n, tests[n].name);
         }
     }
     else
     {
         n = (size_t)atoi(argv[2]);
-        if (n >= 0 && n < _countof(tests))
+        if (n < _countof(tests))
         {
             run_test(n, FALSE);
         }
         else
         {
-            ok(0, "Test out of range: %u\n", n);
+            ok(FALSE, "Test out of range: %Iu (< %Iu)\n", n, _countof(tests));
```

Comment:
Why change `0` to `FALSE`?

Reply:
Only as I find it more explicit. Don't you (want it)?